src/standalone/Vegetation/Canopy.jl

```diff
@@ -1016,6 +1048,9 @@ function ClimaLand.make_update_aux(
         end
         # We update the fa[n_stem+n_leaf] element once we have computed transpiration
 
+        # Update soil moisture stress, used in photosynthesis and conductance
+        update_soil_moisture_stress!(p, Y, canopy.soil_moisture_stress, canopy)
```

Comment:
great! thanks for adding this comment. The order in which we update matters here!